### PR TITLE
Add Durable Object locationHint option

### PR DESCRIPTION
**Pull request by bretthoerner**
Definitely cargo culting here, but the tests seem to pass.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -288,6 +288,21 @@ interface DocumentEnd {
   append(content: Content, options?: ContentOptions): DocumentEnd;
 }
 
+type DurableObjectLocationHint =
+  | "afr"
```
Comment (bcaimano) [inline index.d.ts:292]: Does this work with the initial `|`? I've never seen this before.

**Review by koeninger — APPROVED** [review]

**Review by bcaimano — APPROVED** [review]

**Comment by penalosa** [comment]
Hey! Thanks for the contribution—this repo isn't being maintained any more (we should probably archive it). To change the types, you'll have to submit a PR to https://GitHub.com/Cloudflare/workerd

**Comment by bretthoerner** [comment]
Oops, I'll move it, thanks.

File: index.d.ts (hunk 1 above)
Comment (bretthoerner) [reply index.d.ts:292]: FWIW, yeah. This is result of (mandatory) prettier. :dizzy_face:

**Pull request closed without merging**